cni/client/client.go

```diff
@@ -30,7 +24,7 @@ func New(exec utilexec.Interface) *client {
 
 func (c *client) GetEndpointState() (*api.AzureCNIState, error) {
 	cmd := c.exec.Command(platform.CNIBinaryPath)
-
+	cmd.SetDir(CNIExecDir)
```

Comment:
if CNIExecDir is empty, SetDir will skip setting directory?

Reply:
effectively yes
https://cs.opensource.google/go/go/+/refs/tags/go1.20.5:src/os/exec/exec.go;l=171-174